src/NServiceBus.AcceptanceTesting/Support/ScenarioRunner.cs

```diff
@@ -201,10 +201,13 @@ static void PrintSettings(IEnumerable<KeyValuePair<string, object>> settings)
                     var maxTime = runDescriptor.Settings.TestExecutionTimeout ?? TimeSpan.FromSeconds(90);
                     while (!done() && !cts.Token.IsCancellationRequested)
                     {
-                        if (DateTime.UtcNow - startTime > maxTime)
+                        if (!Debugger.IsAttached)
```

Comment:
at least it doesn't look like the IsAttached property is cached anywhere when I look at it's usages in referencesource.